qemu/tests/cfg/s390x_cpu_model.cfg

```diff
@@ -0,0 +1,45 @@
+- s390x_cpu_model:
+    auto_cpu_model = yes
+    start_vm = yes
+    kill_vm_on_error = yes
+    take_regular_screendumps = no
+    virt_test_type = qemu
+    only s390x
+    variants:
+        - expansion:
+            type = s390x_cpu_model_expansion
+            variants:
+               - model_type_z196_z196_2_z114:
+                    cpu_models = "z196 z196.2 z114"
+                    model_list = ['z196', 'z196.2', 'z114']
```

Comment:
Why do we need both strings and lists?
```
    cpu_models = params.objects('cpu_models')
    for cpu_model in cpu_models:
```
Here we can use the list directly:

`for cpu_model in model_list:`

Reply:
solved